src/ui/color_trigger.ui

```diff
@@ -33,7 +36,8 @@
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
-      <string comment="Ensure that &quot;Default&quot;, &quot;Ignore&quot; and &quot;Cancel&quot; in this instruction are the same as used for the controls elsewhere on this dialog.">&lt;small&gt;Choose:&lt;ul&gt;&lt;li&gt;one of the three ranges below then follow the instructions to select a color from that part of the 256 colors supported&lt;/li&gt;
+      <string comment="Ensure that &quot;Default&quot;, &quot;Ignore&quot; and &quot;Cancel&quot; in this instruction are the same as used for the controls elsewhere on this dialog.">&lt;small&gt;Choose:&lt;ul&gt;&lt;li&gt;one of the basic 16 colors below&lt;/li&gt;
+&lt;li&gt;click the &lt;i&gt;more&lt;/i&gt; button to get access to other colors in the 256-color set, then follow the instructions to select a color from that part of the 256 colors supported; if such a color is already in use then that part will already be showing&lt;/li&gt;
```

Comment:
```html
&lt;small&gt;Choose one of the basic 16 colors below 
or click &lt;i&gt;More colors&lt;/i&gt; to select from all 256 ANSI colors.&lt;/small&gt;
```

1) KISS
2) Yes, there will be instructions, but we don't need to be instructed to follow instructions. 
3) If a users already did this before and selected an advanced color, they will understand their visibility.
4) I feel the "Default" and "Ignore" options need more instructions, but not sure if this is the correct place at all, or only after having clicked them.
5) The "Cancel" button sould be fairly self-explanatory, right?

Too long texts can be overwhelming for regular users and create lots of workload for translation as well.

Reply:
Above comment also references the next few lines which are not shown in github web preview